### ISSUE TITLE
Users:  Usage Report

### ISSUE THREAD

**Issue by jamiemiller1486** [LANResources/amaferm-research-backoffice#15]
Are we tracking user logins anywhere? They are wanting to see how often users are accessing the site.  Would want to see by Role and individual users.

**Comment by reed** [comment]
No, we're not currently tracking logins.

**Comment by jamiemiller1486** [comment]
can we start

**Comment by reed** [comment]
Sure

**Comment by reed** [comment]
We don't expire sessions, so as long as the user doesn't close their browser, they can keep using the site indefinitely without having to sign back in.  So simply keeping track of how many times the user logged in won't be a true representation of usage.  To fix that, I could keep track of the last time the user accessed a page, and when they visit the site, if it's been more than X hours since they last accessed a page, the app would save that as a separate login.  Would that work?

**Comment by jamiemiller1486** [comment]
that's fine.

**Comment by jamiemiller1486** [comment]
Is this in place, someone was asking for some data on users the other day

**Comment by reed** [comment]
No.  I had started it, but then had to stop to work on something else.
